src/main/java/catan/domain/transfer/output/game/DiceDetails.java

```diff
@@ -0,0 +1,41 @@
+package catan.domain.transfer.output.game;
+
+public class DiceDetails {
+    private Integer value;
+    private Integer firstValue;
+    private Integer secondValue;
+
+    public DiceDetails() {
+    }
+
+    public DiceDetails(Integer firstValue, Integer secondValue) {
+        this.value = (firstValue != null ? firstValue + secondValue : null);
```

Comment:
I would avoid logic of "sum value" calculation in different places, why we can't just pass value as constructor parameter?